pkg/cmd/admin/router/router.go

```diff
@@ -594,6 +588,32 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 		if errs := bulk.Create(list, namespace); len(errs) != 0 {
 			return errExit
 		}
+
+		if needsSCC {
+			fmt.Fprintf(out, "Router service account %q is not privileged to open ports on the host IP\n", cfg.ServiceAccount)
```

Comment:
Since this is a warning or error, should be on STDERR and should have prefix "WARNING" (stderr should be cmd.Out()